actionview/lib/action_view/helpers/cache_helper.rb

```diff
@@ -142,13 +143,21 @@ module CacheHelper
       # Works great alongside individual template fragment caching.
       # For instance if the template the collection renders is cached like:
       #
-      #   # notifications/_notification.html.erb
-      #   <% cache notification do %>
+      #   # projects/_project.html.erb
+      #   <% cache project do %>
       #     <%# ... %>
       #   <% end %>
       #
       # Any collection renders will find those cached templates when attempting
       # to read multiple templates at once.
+      #
+      # Similar to normal cache helper, if your template cache depends on multiple sources
+      # you can name all these dependencies as part of an array by passing a callable to `cached`:
```

Comment:
I think I'd just copy from the `cache` docs, and refit a bit:

```ruby
# If your collection cache depends on multiple sources (try to avoid this to keep things simple),
# you can name all these dependencies as part of a block that returns an array:
#
#   <%= render partial: 'projects/project', collection: @projects, cached: -> project { [ project, current_user ] } %>
#
# This will include both records as part of the cache key and updating either of them will
# expire the cache.
```